projects/plugins/jetpack/tests/e2e/lib/pages/wp-admin/recommendations.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Internal dependencies
+ */
+import WpPage from '../wp-page';
+
+const PERSONAL_CHECKBOX_SELECTOR = '.jp-checkbox-answer__container:nth-child(1) input';
```

Comment:
I know there isn't a consistent pattern of how to store and use the selectors across these tests and this approach is much better, but please check `pages/wp-admin/block-editor.js`. This page uses getter methods for selectors and this way they're also encapsulated inside the class. This was proposal that we decided to follow across all pages, but didn't get to do it yet.